ui/components/app/confirmations/alerts/alert-modal/alert-modal.tsx

```diff
@@ -154,7 +158,7 @@ function AcknowledgeCheckbox({
         label={t('alertModalAcknowledge')}
         data-testid="alert-modal-acknowledge-checkbox"
         isChecked={isConfirmed}
-        onClick={() => handleAcknowledgeClick(selectedAlert.key, isConfirmed)}
+        onClick={() => setAlertConfirmed(selectedAlert.key, !isConfirmed)}
```

Comment:
Even If we're passing the function in directly, is it still cleaner to define a `handleCheckboxClick` inside this function for clarity rather than putting logic inline?